task03-one-piece-website/onepiece.html

```diff
@@ -0,0 +1,192 @@
+<!DOCTYPE html>
+<!-- 
+  All Rights Reserved
+  This software is property information of AKUREY S.A.
+  Contact us at contact@akurey.com
+  @summary html of the home section of the one piece website
+-->
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://file.myfontastic.com/KqubBRuJG8iZrRQgJ3mBeb/icons.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;300;400;700&family=Roboto:wght@100;300;400;700&display=swap" rel="stylesheet">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
+    <link rel="stylesheet" type="text/css" href="../task03-one-piece-website/css/onepiece.css">
+    <script src="onepiece.js"></script>
+    <title>One Piece</title>
+</head>
+<body>
+    <header>
+          <nav class="navbar fixed-top navbar-expand-xl navbar-light bg-white py-4 justify-content-center ">
+            <button  class="navbar-toggler burger-menu" type="button" data-toggle="collapse" data-target="#navbarTogglerDemo01" aria-controls="navbarTogglerDemo01" aria-expanded="false" aria-label="Toggle navigation">
+              <span id="burger-icon" class="icon-align-left"></span>
+              <span id="burger-icon" class="icon-times"></span>
+            </button>
+            <div class="logo">
+              <img class="one-piece-logo"src="../images/one-piece-logo.png" alt="onepiece">
```

Comment:
src doesn't have a space after the class.
Also, give alt a significant name, like "One Piece logo" or some text that represents that image with more clarity